camayoc/tests/qcs/test_credentials.py

```diff
@@ -167,11 +254,14 @@ def test_negative_create_no_key_or_pass(cleanup):
     :expectedresults: Error is thrown and no new host credential is created.
     :caseautomation: notautomated
```

Comment:
`:caseautomation` needs updated

(I'm mostly marking these so when it gets updated, these comments will get squashed and it will be easy to see that they are all fixed)

Reply:
Thank you, will fix all.